Deployment/templates/continous-deployment.yml

```diff
@@ -0,0 +1,31 @@
+parameters:
+  - name: terraformapply
+    type: string
```

Comment:
I think type of this should be Boolean.

Reply:
Changed param name to ContinueEvenIfResourcesAreGettingDestroyed